Base.rte/Scenes/Objects/Bunkers/BunkerSystems/Movators/GlobalMovatorFunctions.lua

```diff
@@ -0,0 +1,160 @@
+if not MovatorData then
+	MovatorData = {};
+	for team = Activity.NOTEAM, Activity.TEAM_4 do
+		MovatorData[team] = {
+			energyLevel = 0,
+			nodeData = {},
+			nodeDataCount = 0,
+			teleporterNodes = {},
+			teleporterNodesCount = 0,
+		};
+	end
+	print("Setup movator data")
+end
+
+local targetNodeIsNotObstructed = function(startNode, targetNode, direction, nodeZoneSize)
+	local checkWrapping = SceneMan.SceneWrapsX or SceneMan.SceneWrapsY;
+
+	local spreadRaysHorizontally = direction == Directions.Up or direction == Directions.Down;
+	local rayOffsets = {
+		spreadRaysHorizontally and Vector(-nodeZoneSize.X * 0.25, 0) or Vector(0, -nodeZoneSize.Y * 0.25),
+		Vector(),
+		spreadRaysHorizontally and Vector(nodeZoneSize.X * 0.25, 0) or Vector(0, nodeZoneSize.Y * 0.25),
+	};
+
+	local rayVector = SceneMan:ShortestDistance(startNode.Pos, targetNode.Pos, checkWrapping);
+	for _, rayOffset in ipairs(rayOffsets) do
+		if SceneMan:CastStrengthRay(startNode.Pos + rayOffset, rayVector, 15, Vector(), 4, 0, true) then
+			return false;
+		end
+	end
+	return true;
+end
+
+Movators_CheckConnections = function(node)
```

Comment:
`function Movators_CheckConnections(node)` should be identical syntax sugar for this same thing, and since the syntax sugar is way more common in other Lua files I'd consider using it instead. Though there may be a good reason to keep it like this that I am not seeing right now?

Reply:
Oops, yeah. I did it this way cause the non-global functions need to be written this way, but you're right that there's no need to for global ones. Thanks

Reply:
Oooh good point!